src/v2i-hub/TelematicBridgePlugin/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+PROJECT (TelematicBridgePlugin VERSION 7.5.1 LANGUAGES CXX)
+
+set (TMX_PLUGIN_NAME "Telematic Bridge")
+
+find_package(RapidJSON REQUIRED)
+
+BuildTmxPlugin()
+TARGET_LINK_LIBRARIES ( ${PROJECT_NAME} tmxutils jsoncpp)
```

Comment:
if we are not linking rapidjson, what is the point of finding the package?

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/565/files#diff-44401677ab8e59119c4870aabc314fba94c3a4cb82288919cf11b36257a5efe0